bazel/go_proto_library.bzl.diff

```diff
@@ -1,174 +0,0 @@
-diff --git a/usr/local/google/home/htuch/src/rules_go/proto/go_proto_library.bzl b/go_proto_library.bzl
```

Comment:
How come you're removing this? From the commit message, it sounds like you played around with making this better, but it didn't work, why remove the existing support, or is that broken?

Reply:
The `go_proto_library.bzl` is still there just removed these diff files. They seemed to be accidental or am I missing something?

Reply:
They're intended to provide a summary of where we've diverged against the base proto libs we forked. I'd leave them there for now. I agree we need to do some more work on improving the Go proto build, will poke at this after this PR is in.

Reply:
Ah ok i'll put em back

Reply:
Restored!